drivers/counter/Kconfig.rv3032_counter

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2025 BayLibre, SAS
+# SPDX-License-Identifier: Apache-2.0
+
+config COUNTER_RV3032
+	bool "RV3032 Counter driver"
+	default y
+	depends on DT_HAS_MICROCRYSTAL_RV3032_COUNTER_ENABLED
+	depends on MFD_RV3032
```

Comment:
This should be a select

Reply:
That's what I want to achieve with the select, the RTC, sensor, or counter, require the parent to be selected, so if you try to enable only one of the child devices, but the parent is not enabled (status="disabled" or MFD=n), that is a bug, its a direct dependency.